### PR TITLE
Boards, sprints and issues from API only

### PR DESCRIPTION
**Pull request by VincentCATILLON**

### CODE REVIEW

File: Api/JiraBoardConfiguration.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ * Jira board configuration
+ * @author Vincent Catillon <contact@vincent-catillon.fr>
+ */
+class JiraBoardConfiguration extends JiraConfiguration
+{
+    /**
+     * Api URI
+     * @var string $api
+     */
+    protected $api = '/rest/agile/1.0';
```
Comment (ooga) [inline Api/JiraBoardConfiguration.php]: ```/rest/agile/latest``` ?

File: Service/JiraService.php [hunk 2]
```diff
@@ -4,97 +4,196 @@
...
+     */
+    public function getSprints()
     {
-        return $this->sprintId;
+        $config = new JiraBoardConfiguration();
+        $values = array();
+        if (!empty($this->boardId)) {
+            $config->setBoardId($this->boardId);
+            $config->setParameters(array(
+                'state' => 'active'
```
Comment (ooga) [inline Service/JiraService.php]:
```
'state' => 'active',
'state' => 'future'
```

**Comment by VincentCATILLON** [comment]
Nice code review :+1: 
This PR is up too date

File: Api/JiraBoardConfiguration.php (hunk 1 above)
Comment (VincentCATILLON) [inline Api/JiraBoardConfiguration.php]: Yes, it's an overrided property needed to get boards results from Jira API.

**Comment by ooga** [comment]
need rebase

**Comment by ooga** [comment]
need rebase

**Comment by brieucpouliquen** [comment]
duplicate of #https://github.com/ooga/ScrumBoard-it/pull/23

**Pull request closed without merging**